css/src/lib.rs

```diff
@@ -109,22 +105,19 @@ const RESERVED3_LEN: usize = 32;
 const RESERVED4_LEN: usize = 12;
 
 /// The SIGSTRUCT structure
-#[repr(C, packed)]
-#[derive(Clone)]
+#[derive(Clone, Debug, Eq, Hash, Ord, PartialEq, PartialOrd)]
```

Comment:
Since the logic now uses nom the repr c packed seemed like more of a hindrance than a help